sqlx-macros-core/src/database/impls.rs

```diff
@@ -44,7 +44,7 @@ mod sqlx {
     #[cfg(feature = "postgres")]
     pub use sqlx_postgres as postgres;
 
-    #[cfg(feature = "sqlite")]
+    #[cfg(any(feature = "sqlite", feature = "sqlite-unbundled"))]
```

Comment:
To avoid having to do this everywhere, both these features could enable a feature like `_sqllite` or `_sqlite-base`. We use the underscore prefix to indicate a feature that's not meant for direct use.